src/formatter/index.ts

```diff
@@ -23,11 +20,7 @@ export interface FormatOptions {
   [customKey: string]: any
 }
 
-export type IFormatterStream =
-  | FsWriteStream
-  | TtyWriteStream
-  | PassThrough
-  | HttpStream
+export type IFormatterStream = Writable
```

Comment:
`HttpStream` was no longer applicable here but also this didn't feel like a useful type. We need a writable stream, is all.